diff --git a/Sources/QtBridgeMacros/Extensions.swift b/Sources/QtBridgeMacros/Extensions.swift
--- a/Sources/QtBridgeMacros/Extensions.swift
+++ b/Sources/QtBridgeMacros/Extensions.swift
@@ -124,7 +124,7 @@ extension TypeSyntax {
     }
 
     var isSupportedReturnType: Bool {
-        return isSupportedSettableType || isVoid
+        return isSupportedSettableType || isVoid || self.is(IdentifierTypeSyntax.self)
     }
 
     var isVoid: Bool {
diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -62,6 +62,7 @@ ExternalProject_Add(QtBridgeMacros_External
         -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
         -DBUILD_SHARED_LIBS=YES
         -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=${QTBRIDGE_MACROS_EP_ROOT}/build/bin
+    BUILD_ALWAYS TRUE
     USES_TERMINAL_BUILD TRUE
     INSTALL_COMMAND ""
 )
